Geometry/GEMGeometryBuilder/src/GEMGeometryBuilderFromDDD.cc

```diff
@@ -54,13 +54,14 @@ GEMGeometry* GEMGeometryBuilderFromDDD::build(const DDCompactView* cview, const
 
 GEMGeometry* GEMGeometryBuilderFromDDD::buildGeometry(DDFilteredView& fview, const MuonDDDConstants& muonConstants)
 {
-  std::cout << "Building the geometry service" << std::endl;
   LogDebug("GEMGeometryBuilderFromDDD") <<"Building the geometry service";
   GEMGeometry* geometry = new GEMGeometry();
 
   LogDebug("GEMGeometryBuilderFromDDD") << "About to run through the GEM structure\n" 
 					<<" First logical part "
 					<<fview.logicalPart().name().name();
+
+  std::map<std::string,std::tuple<int,float,float,float,float> > chamberPar;
```

Comment:
@bsunanda - ok, but just to clarify, you do you or do you not know the set of expected strings ahead of time?